src/main/kotlin/com/bugsnag/android/gradle/internal/GradleUtil.kt

```diff
@@ -0,0 +1,92 @@
+@file:Suppress("MatchingDeclarationName") // This file contains multiple top-level members
+package com.bugsnag.android.gradle.internal
```

Comment:
I mentioned this to @fractalwrench separately, but I noticed that the plugin's regular package is actually one file with dots in the name

![image](https://user-images.githubusercontent.com/1361086/89756392-56962080-dab0-11ea-9b42-692cc544bfca.png)

Reply:
I've raised an internal ticket for this and will address this once the open PRs have been merged to avoid any potential for merge conflicts.